docs/framework/migration-guide/toc.yml

```
@@ -1,28 +1,246 @@
-- name: Migration Guide
+- name: Migration guide
   href: index.md
+- name: Application compatibility
+  href: application-compatibility.md
   items:
-  - name: Application Compatibility
-    href: application-compatibility.md
+  - name: Runtime changes
     items:
-    - name: Runtime Changes
-      href: runtime/index.md
-    - name: Retargeting Changes
-      href: retargeting/index.md
-  - name: Version Compatibility
-    href: version-compatibility.md
-  - name: Versions and Dependencies
+    - name: Version 4.0 to version 4.5
```

should we simplify this? similar to https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?

```suggestion
    - name: 4.0 to 4.5
```

---

Done.